CodeStarterCamp_Week1/main.swift

```diff
@@ -3,91 +3,83 @@ import Foundation
 drawIceTop()
 drawIceBar()
 
+func drawPeperoBarLength(BarLength: Int) {
+    for _ in 1...BarLength {
+        print(" | | ")
+    }
+}
+
+func drawPeperoBody(body: String, topping: String) {
+    print(topping + body + topping)
+}
+
+func drawPeperoBar(lenght: Int, topping: String, body: String) {
+    if body == "***" {
+        for _ in 1...lenght {
+            drawPeperoBody(body: body, topping: topping)
+        }
+    }
+    else {
+        for i in 1...lenght {
+            if (i % 2) != 0{
```

Comment:
```suggestion
            if (i % 2) != 0 {
```
괄호를 여실 땐, 한 칸 띄워주세요!

`(i % 2) != 0` 는 `i` 가 홀수인지 여부를 체크하는 거죠?
계산이 들어가는 코드가 if 문의 조건으로 들어갈 땐, 상수에 조건을 할당해주어 `(i % 2) != 0` 라는 코드가 어떤 의미인지를 표현해주면, 다른 사람이 코드를 볼 때 한 번 더 생각하지 않고 직관적으로 어떤 조건인지 이해할 수 있어요!